src/sentry/api/serializers/models/organization_member.py

```diff
@@ -34,3 +36,40 @@ def serialize(self, obj, attrs, user):
             'dateCreated': obj.date_added,
         }
         return d
+
+
+class OrganizationMemberWithTeamsSerializer(OrganizationMemberSerializer):
+    def get_attrs(self, item_list, user):
+        attrs = super(OrganizationMemberWithTeamsSerializer,
+                      self).get_attrs(item_list, user)
+
+        member_team_map = list(OrganizationMemberTeam.objects.filter(
+            organizationmember__in=item_list).values(
+            'organizationmember', 'team'))
```

Comment:
This needs to only fetch `team_id`, not the full `team`. And below, use `team_id` as well. Otherwise, you're piling on a bunch of extra queries.

Reply:
Should we also do the same thing for `organizationmember`?

Reply:
I just tested this locally... I guess it doesn't resolve the nested object when using `values()`. This simply returns the same thing as `team_id` or `organizationmember_id`.

I didn't know this. I don't care if you change it now, but I prefer to always be explicit when I'm trying to work with ids instead of full objects, but that's just my preference. In this case, there is no difference in behavior.